kura/org.eclipse.kura.net.configuration/src/main/java/org/eclipse/kura/net/configuration/NetworkConfigurationServiceCommon.java

```diff
@@ -108,6 +108,15 @@ public static Tocd getDefinition(Map<String, Object> properties, Optional<List<U
                     getDnsDefinition(tocd, ifaceName);
                     getModemDefinition(tocd, ifaceName);
                     break;
+                case VLAN:
+                    if (usbNetDevices.isPresent()) {
+                        getUsbDeviceDefinition(usbNetDevices.get(), tocd, ifaceName);
+                    }
```

Comment:
Sonar is whining [about Cognitive Complexity](https://sonarcloud.io/project/issues?resolved=false&severities=CRITICAL&types=CODE_SMELL&branch=PR-4816&id=org.eclipse.kura%3Akura&open=AYooGPqpeo7TZ9t3jvzn) here.

I'll teach you a simple trick to make it shut up: if you substitute each `if` statement with a `ifPresent` call it won't add to the cognitive complexity computation 🤫 😉 

For example:

```suggestion
                    usbNetDevices.ifPresent(usbNetDevice -> getUsbDeviceDefinition(usbNetDevice, tocd, ifaceName));
```

This should take care of this Critical Issue